File: problem2/fixed/B01074900.py
def count_pairs(nums, T):



    left = 0
    right = len(nums)-1
    sum = 0


    while left <= right:

        if (nums[left] + nums[right]) <= T:
            sum = (right - left)
            left += 1
        elif (nums[left] + nums[right]) > T:
            right -= 1


    return sum

File: problem2/java/B01074900.java
public class Solution {
public static int countPairs(int[] nums, int T) {
    // Paste this into your answer if you want to use Java

   //Set up two pointers
   int left = 0;
   int right = nums.length-1;
   int sum = 0;

   //Check for pairs
   while(left <= right){

      if((nums[left] + nums[right]) <= T){
            sum = (right - left);
            left++;
      }
      else if((nums[left] + nums[right]) > T){
            right--;
      }

   }

   return sum;

}
}